src/Microsoft.Health.Dicom.Core/Extensions/DicomMediatorExtensions.cs

```diff
@@ -281,13 +282,17 @@ public static class DicomMediatorExtensions
     }
 
     public static Task<UpdateWorkitemResponse> UpdateWorkitemAsync(
-        this IMediator mediator, Stream requestBody, string requestContentType, string workitemInstanceUid, string transactionUid = default, CancellationToken cancellationToken = default)
+        this IMediator mediator,
+        DicomDataset dicomDataset,
+        string requestContentType,
+        string workitemInstanceUid,
+        string transactionUid = default,
+        CancellationToken cancellationToken = default)
     {
         EnsureArg.IsNotNull(mediator, nameof(mediator));
-        EnsureArg.IsNotNull(requestBody, nameof(requestBody));
```

Comment:
Don't we still care about `null`-checking?

Reply:
It is validated in the API pipeline, as well as in the Handler/Validator.